lib/SiftRequest.php

```diff
@@ -99,12 +100,20 @@ public function send() {
         // Send the request using curl and parse result
         curl_setopt($ch, CURLOPT_HTTPHEADER, $headers);
         $result = curl_exec($ch);
-        $httpStatusCode = curl_getinfo($ch, CURLINFO_HTTP_CODE);
 
-        // Close the curl connection
-        curl_close($ch);
+        // Check for curl errors and/or request failure
+        $curl_errno = curl_errno($ch);
+        if($result == false && $curl_errno) {
+          $this->error = "Connection Error Code ".curl_errno($ch)." - ".curl_error($ch);
+          curl_close($ch);
+          return new SiftResponse($result, 999, $this);
```

Comment:
I'd use `0` or `NULL` for the placeholder. Someone might have `if (code >= 500)` somewhere.

Reply:
Got it. Fair point. I wasn't super excited by the 999 in the first place, so that's more than fine to remove.